x-pack/plugins/uptime/public/components/certificates/cert_status.tsx

```diff
@@ -15,20 +17,39 @@ interface Props {
   cert: Cert;
 }
 
+const DateText = styled(EuiText)`
+  display: inline-block;
+  margin-left: 5px;
+`;
+
 export const CertStatus: React.FC<Props> = ({ cert }) => {
   const certStatus = useCertStatus(cert?.not_after, cert?.not_before);
 
+  const relativeDate = moment(cert?.not_after).fromNow();
+
   if (certStatus === CERT_STATUS.EXPIRING_SOON) {
     return (
       <EuiHealth color="warning">
-        <span>{labels.EXPIRES_SOON}</span>
+        <span>
+          {labels.EXPIRES_SOON}
+          {'  '}
+          <DateText color="subdued" size="xs">
+            {relativeDate}
+          </DateText>
+        </span>
       </EuiHealth>
     );
   }
   if (certStatus === CERT_STATUS.EXPIRED) {
     return (
       <EuiHealth color="danger">
-        <span>{labels.EXPIRED}</span>
+        <span>
+          {labels.EXPIRED}
+          {'  '}
+          <DateText color="subdued" size="xs">
+            {relativeDate}
+          </DateText>
+        </span>
       </EuiHealth>
     );
   }
```

Comment:
Was it a deliberate choice to omit the relative date from `CERT_STATUS.TOO_OLD` case?

Reply:
Example: 
![image](https://user-images.githubusercontent.com/18429259/83680565-22354e00-a5af-11ea-9159-e3d405f2bb01.png)

Reply:
i took care of it.